src/utils/mutations.ts

```diff
@@ -900,5 +1122,6 @@ export async function addComment(
 		content: comment.content,
 		likes: 0,
```

Comment:
As in another comment, it may be better to not store likes and instead just calculate likes based on length of likes client side